runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/stores/AbstractBlockStore.java

```diff
@@ -28,14 +29,18 @@
  */
 public abstract class AbstractBlockStore implements BlockStore {
   private final SerializerManager serializerManager;
+  private final MemoryPoolAssigner memoryPoolAssigner;
 
   /**
    * Constructor.
    *
    * @param serializerManager the coder manager.
+   * @param memoryPoolAssigner the memory pool assigner.
    */
-  protected AbstractBlockStore(final SerializerManager serializerManager) {
+  protected AbstractBlockStore(final SerializerManager serializerManager,
```

Comment:
Remove the `protected` modifier here and others (i.e., use no modifier). My Intellij recommends it. 

See https://stackoverflow.com/a/215505